tests/e2e/kubetest2-kops/deployer/up.go

```diff
@@ -308,11 +330,19 @@ func (d *deployer) zones() ([]string, error) {
 }
 
 // appendIfUnset will append an argument and its value to args if the arg is not already present
-// This shouldn't be used for arguments that can be specified multiple times like --set
+// This shouldn't be used for arguments that can be specified multiple times except --set
 func appendIfUnset(args []string, arg, value string) []string {
+	setFlags := []string{}
 	for _, existingArg := range args {
 		existingKey := strings.Split(existingArg, "=")
```

Comment:
There are cases where there are even more "=" in the expression.
```suggestion
		existingKey := strings.SplitN(existingArg, "=", 2)
```